buildSrc/src/main/groovy/bio.terra.catalog.java-common-conventions.gradle

```
@@ -83,8 +83,10 @@ spotbugs {
 }
 spotbugsMain {
     reports {
-        sarif {
-            enabled = true
+        if (isCiServer) {
+            sarif.enabled = true
+        } else {
+            html.enabled = true
```

There's probably no downside to always emit both forms in both cases. 
```suggestion
        sarif.enabled = true
        html.enabled = true
```

Also I've been thinking that we can spotbugs completely as I believe sonarqube includes spotbugs as part of its static analysis. But I haven't verified this theory yet...

---

Unfortunately you can't generate `html` and `sarif` reports simultaneously.